tests/integration/StorageFileTest.php

```diff
@@ -14,10 +14,12 @@ final class StorageFileTest extends TestCase
 	public function setup(): void
 	{
 		parent::setUp();
-		$authHeader = ['Authorization' => 'Bearer '.'<service_role>'];
-		$bucket_id = '<existing-storage-bucket>';
-		$this->client = new  \Supabase\Storage\StorageFile('https://<project_ref>.supabase.co/storage/v1',
-			$authHeader, $bucket_id);
+		$dotenv = \Dotenv\Dotenv::createUnsafeImmutable(dirname(__DIR__, 2), '.env.test');
```

Comment:
We should use a path here e.g. __DIR__ . '/../../env.test'; vs dirname()

Reply:
Done